base/cvd/cuttlefish/host/commands/cvd/cli/commands/reset.cpp

```diff
@@ -158,6 +158,11 @@ class CvdResetCommandHandler : public CvdCommandHandler {
     }
     CF_EXPECT(KillAllCuttlefishInstances(
         /* clear_instance_dirs*/ options.clean_runtime_dir));
+
+    if (DirectoryExists("/tmp/acloud_cvd_temp/")) {
+      CF_EXPECT(RecursivelyRemoveDirectory("/tmp/acloud_cvd_temp/"));
```

Comment:
Does it seem worth it to delete the contents of this directory without deleting the directory itself?

I could imagine a user changing the directory permissions of this to facilitate some multi-user interaction, though it would have to be reset on every reboot anyway since `/tmp` is not going to be persistent. Maybe not that significant in that context.